src/ui/public/kbn_share/kbn_share.less

```diff
@@ -0,0 +1,42 @@
+.sharing-ui-menu__List {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.sharing-ui-menu__ListWrapper,
+.sharing-ui-menu__SharePanel {
```

Comment:
Hmmm... I think you should try treating the component as the namespace. For example, `shareMenu`:

```less


/**
 * The shareMenu has a lot of children, indicated by the indented classes
 * below it.
 */
.shareMenu {
  text-align: center;
  display: inline-block;
  vertical-align: top;
}

  /**
   * This single indent indicates the child relationship. This class is tightly
   * coupled to the shareMenu. It literally can't be used outside of the
   * parent element... it just wouldn't make any sense in the UI. It has
   * the double-undescores because the shareMenu can't work without it either.
   */
  .shareMenu__list {
    display: block;
    margin: 0;
    padding: 0;
  }

    /**
     * This shareMenuItem is indented beneath the list for the same reason.
     * Its naming pattern changes because it's going to have child elements
     * and have something like .shareMenu__list__item__label__text will become
     * too verbose. This component has a little more identity that shareMenu__list
     * so it seems like a good place to re-establish its role as a parent of
     * child components, via its name.
     */
    .shareMenuItem {
      display: inline-block;
      text-align: center;
      padding: 10px;
    }

      .shareMenuItem__icon {
        display: block;
        font-size: 28px;
      }

/**
 * This is non-indented, because it's not a child of shareMenu. They're related
 * via the 'share' in their name, but it's clear they have different roles and
 * their relationship is distant, not direct.
 */
.sharePanel {
  display: inline-block;
  vertical-align: top;
  padding: 10px;
  width: ~"calc(100% - 150px)";
}
```

Want to Zoom about this tomorrow?